app/data_access/user.py

```diff
@@ -335,39 +331,11 @@ def additional_activity_filters(query):
         return to_connection(
             missions,
             connection_cls=MissionConnection,
-            get_cursor=lambda m: f"{str(m.activities_for(self)[0].start_time)},{m.id}",
+            get_cursor=lambda m: f"{str(m.activities_for(self, include_dismissed_activities=include_deleted_missions)[0].start_time)},{m.id}",
```

Comment:
:orange_circle: Je ne suis toujours pas sûre de ce comportement. Cela veut dire que pour une mission non supprimée, on souhaite remonter les activités dismissed dès lors qu'on veut afficher les missions supprimées. Ce comportement caché ne me semble pas clair. J'aurai laissé ici la valeur par défaut, comme avant : 
```suggestion
            get_cursor=lambda m: f"{str(m.activities_for(self)[0].start_time)},{m.id}",
```